src/server/game/Grids/Notifiers/GridNotifiers.h

```diff
@@ -1321,10 +1322,18 @@ namespace Trinity
             }
 
         private:
-            WorldObject const& i_obj;
-            uint32 i_entry;
-            bool   i_alive;
-            float  i_range;
+            bool IsPersonalSpawnOfAnoter(Creature* creature)
+            {
+                if (TempSummon const* tempSummon = creature->ToTempSummon())
+                    if (tempSummon->IsVisibleBySummonerOnly() && i_obj.GetGUID() != tempSummon->GetSummonerGUID())
+                        return true;
+                return false;
+            }
+
+            WorldObject const&      i_obj;
+            uint32                  i_entry;
+            bool                    i_alive;
+            float                   i_range;
```

Comment:
Looks useful & tidy. I'm sort of torn regarding these elements and whether the tidy format follows TC codestyle (I can't remember seeing this arrangement before).

Reply:
It was used before the chageI made but just on the 3 variables rather than the 4.
It's also used extensively in the code, although mostly in enums. I don't know if it's the current standard since I believe it was changed over the years, but here are few examples where it's used:

- https://github.com/TrinityCore/TrinityCore/blob/master/src/server/database/Database/MySQLConnection.h#L101
- https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/Battlegrounds/Zones/BattlegroundAB.h#L359
- https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/Entities/Creature/Creature.h#L441

Reply:
The current standard is "as few changes as possible"